lib/superstore/attribute_methods/primary_key.rb

```diff
@@ -4,7 +4,7 @@ module PrimaryKey
       extend ActiveSupport::Concern
 
       included do
-        include ActiveRecord::AttributeMethods::PrimaryKey
+        # include ActiveRecord::AttributeMethods::PrimaryKey
```

Comment:
Better to delete than comment this out.

Reply:
f1bcdffa836fdbb391f281b21ebb9232bec23297